### ISSUE TITLE
[YYC] Using chained accessor in struct throws "I32 argument is unset" error

### ISSUE THREAD

**Issue by shichen85** [YoYoGames/GameMaker-Bugs#4522]
### Description
We noticed that the 2024 runtime beta started crashing when using  chained accessor in struct for YYC export. This was not the case for the stable release 2023.11.1.160.

The testing code is:
```
var s = {a: {hello: { lines : [  ], console : 1, verbose : 1, level : 2, enabled : 1 }}}
var tag = "hello";
s.a[$ tag].level = 1;
```

### Steps To Reproduce
1. Run the sample in runtime  2023.11.1.160 in both YYC and VM. Notice no crash.
2. Run in runtime 2024.200.0.490. Notice that VM does not crash, but YYC crashes

[6d8f078f-dd05-456b-8fc3-c4eafb309e83](https://api.gamemaker.io/api/github/downloads/6d8f078f-dd05-456b-8fc3-c4eafb309e83)

**Comment by shichen85** [comment]
May relate to https://github.com/YoYoGames/GameMaker-Bugs/issues/4509

**Comment by rwkay** [comment]
Fix for #4509 has fixed this one too.

**Comment by scott-dunbar** [comment]
Verified fixed in IDE v2024.200.0.476  Runtime v2024.200.0.491
